packages/pages/package-lock.json

```diff
@@ -4762,7 +4762,8 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true
+          "bundled": true,
+          "optional": true
```

Comment:
Please remove this changes, they are introduced by  `npm` but are not relevant for this PR.

Reply:
Still relevant.

Reply:
Still not fixed.